packages/contentstack-migration/src/utils/migration-logger.js

```diff
@@ -0,0 +1,28 @@
+const winston = require('winston');
+const path = require('path');
+module.exports = class MigrationLogger {
+  constructor(filePath) {
+    this.filePath = path.join(filePath, 'migration-logs');
+    this.logger = winston.createLogger({
+      levels: { info: 1, error: 2 },
+      transports: [
+        new winston.transports.File({
+          level: 'error',
+          filename: path.join(this.filePath, 'error.logs'),
+          format: winston.format.combine(winston.format.timestamp(), winston.format.json()),
+        }),
+        new winston.transports.File({
+          level: 'info',
+          filename: path.join(this.filePath, 'info.logs'),
+          format: winston.format.combine(winston.format.timestamp(), winston.format.json()),
+        }),
+      ],
```

Comment:
@cs-raj , if info not required then please remove it

Reply:
Removed it and kept error logger only